content/en/docs/ops/diagnostic-tools/controlz/index.md

```diff
@@ -14,23 +14,22 @@ Istio components are built with a flexible introspection framework which makes i
 of a running component. Components open a port which can be used from a web browser to get an interactive view into the state of the
 component, or via REST for access and control from external tools.
 
-Mixer, Pilot, and Galley all implement the ControlZ functionality. When these components start, a message is logged indicating the
+Istiod implement the ControlZ functionality. When the component start, a message is logged indicating the
 IP address and port to connect to in order to interact with ControlZ.
 
 {{< text plain >}}
-2018-07-26T23:28:48.889370Z     info    ControlZ available at 100.76.122.230:9876
+2020-08-04T23:28:48.889370Z     info    ControlZ available at 100.76.122.230:9876
 {{< /text >}}
 
 Here's sample of the ControlZ interface:
 
-{{< image width="80%" link="./ctrlz.png" caption="ControlZ User Interface" >}}
+{{< image width="90%" link="./ctrlz.png" caption="ControlZ User Interface" >}}
 
-To access the ControlZ page of deployed components (i.e. Mixer, Galley, Pilot), you can port-forward their ControlZ endpoints
+To access the ControlZ page of deployed component (i.e. istiod), you can port-forward their ControlZ endpoints
 locally and connect through your local browser:
 
 {{< text bash >}}
-$ istioctl dashboard controlz <podname>
+$ istioctl dashboard controlz istiod-954c974bb-qgb4h -n istio-system
```

Comment:
```suggestion
$ istioctl dashboard controlz <istiod-podname> -n istio-system
```
@ericvn now that we support `snip_id=none`, we can use it for snippets with invalid bash syntax, e.g., `<podname>`, instead of removing the `<` and `>`. Since this kind of example is not a runnable command, as is, there's no need to generate the snip anyway.

Reply:
@frankbu Is it ok to name it like `istiod-<podname>` as `istiod` will not change?

Reply:
@shamsher31 That's not quite right, because the `<...>` part would not be a podname, but rather a pod name suffix. Maybe a better (less confusing) syntax would be `<istiod pod name>`?

Reply:
Updated suggestion above

Reply:
Thanks